storage/badger/headers_test.go

```diff
@@ -50,3 +50,22 @@ func TestHeaderRetrieveWithoutStore(t *testing.T) {
 		require.True(t, errors.Is(err, storage.ErrNotFound))
 	})
 }
+
+func TestTimeStampRetrieve(t *testing.T) {
```

Comment:
this test is currently failing since there the `blockID-> timestatmp` index is not being updated.